docs/licensing-policy.md

```diff
@@ -0,0 +1,47 @@
+# IPFS Licensing Policy
+
+In order to ensure an open source project is actually open and re-usable in a legal sense, it’s critical that licensing information is clear. Repositories in the `ipfs`, `libp2p`, and `ipld` organizations should all follow this policy for licensing their contents.
+
+Unless there is a particular reason to use a different licensing structure, please use the following for new projects:
+
+1. Copyright should always be assigned to **Protocol Labs, Inc,** regardless of the type of license. (In an ideal future, we can assign copyright to a foundation, but Protocol Labs is the best usable legal entity we have now.)
+
+2. All software code should be licensed under the MIT license. Canonical license text for the MIT license can be found in this repository’s [`LICENSE` file](../LICENSE) or at the [Open Source Initiative website](https://opensource.org/licenses/MIT).
+
+2. All documentation and non-code resources (e.g. logos and imagery) should be licensed under the Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0) license. An [easy-to-read, linkable description](https://creativecommons.org/licenses/by-sa/4.0/) and canonical license text in [HTML](https://creativecommons.org/licenses/by-sa/4.0/legalcode) and [plain text](https://creativecommons.org/licenses/by-sa/4.0/legalcode.txt) can be found at the Creative Commons website.
```

Comment:
I'd be in favour of just CC-BY as it matches MIT well, it's without copyleft.

Reply:
I don’t have any particular opinion, but that certainly makes sense. It would be great to hear from more people which is preferable.

Reply:
Ok, I updated this to CC-BY since there was no other input.